packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -1213,27 +1246,62 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
     }
 
     private onKeyDown(event: _ModuleSupport.KeyInteractionEvent<'keydown'>) {
+        const { state } = this;
+        const context = this.getAnnotationContext();
+
         const { sourceEvent } = event;
         const { shiftKey } = sourceEvent;
         const modifierKey = sourceEvent.ctrlKey || sourceEvent.metaKey;
 
         this.state.transition('keyDown', { shiftKey });
 
-        if (modifierKey && sourceEvent.key === 'c') {
-            this.state.transition('copy');
-        } else if (modifierKey && sourceEvent.key === 'x') {
-            this.state.transition('cut');
-            this.recordActionAfterNextUpdate('Cut annotation');
-        } else if (modifierKey && sourceEvent.key === 'v') {
-            this.state.transition('paste');
-            this.recordActionAfterNextUpdate('Paste annotation');
+        const translation = { x: 0, y: 0 };
+
+        const xStep = Math.max(context?.xAxis.scaleBandwidth() ?? 0, modifierKey ? 10 : 1);
+        const yStep = Math.max(context?.yAxis.scaleBandwidth() ?? 0, modifierKey ? 10 : 1);
```

Comment:
This should be on ctrl or shift, not the meta key.

Reply:
good spot